.vib/redis/goss/vars.yaml

```diff
@@ -0,0 +1,16 @@
+binaries:
+  - gosu
+  - redis-benchmark
+  - redis-cli
+  - redis-server
+  - wait-for-port
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/redis/data
+      - /opt/bitnami/redis/etc
```

Comment:
I think there are some inconsistencies here compared to the permissions modifications and folder creations done at the postunpack stage. Could you please review this?

Reply:
I've added the paths of the folders that are created in the postunpack stage.  Although I'm not sure what you mean with inconsistencies in permission modifications.